modules/oe_content_event/config/install/field.field.node.oe_event.field_registration_end_date.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_registration_end_date
+    - node.type.oe_event
+  module:
+    - datetime
+id: node.oe_event.field_registration_end_date
+field_name: field_registration_end_date
+entity_type: node
+bundle: oe_event
+label: 'Registration end date'
+description: ''
+required: false
```

Comment:
Should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Report fields group filled in.